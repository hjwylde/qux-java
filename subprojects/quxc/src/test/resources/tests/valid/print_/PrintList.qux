void main([str] args) {
    print [true, false];
    print [null, true, null];
    print [0, [null, 0], 0.0, 'foo'];
}

