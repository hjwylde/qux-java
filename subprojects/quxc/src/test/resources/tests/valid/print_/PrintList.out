[true, false]
[null, true, null]
[0, [null, 0], 0.0, foo]
